### DISCUSSION TITLE
abotu compile corvus

### DISCUSSION THREAD

**Discussion by donboruza** [ROM-builders/temporary#12104] [Q&A]
sir,I wanna ask about compile corvus . 
its source issue or what . 
when I try corvus its always failed in sync . 
its maybe bcs superproject or what?
or its not yet supported on cirrus?

**Comment by Apon77** [comment]
> sir,I wanna ask about compile corvus . its source issue or what . when I try corvus its always failed in sync . its maybe bcs superproject or what? or its not yet supported on cirrus?
no idea, it was supported afaik.
